{RepositoryName}/src/{Namespace}.Web/Scripts/module.js

```diff
@@ -0,0 +1,42 @@
+// Call this to register your module to main application
+var moduleName = '{ModuleId}';
+
+if (AppDependencies !== undefined) {
+    AppDependencies.push(moduleName);
+}
+
+angular.module(moduleName, [])
+    .config(['$stateProvider', '$urlRouterProvider',
+        function ($stateProvider, $urlRouterProvider) {
+            $stateProvider
+                .state('workspace.{ModuleId}State', {
+                    url: '/{ModuleId}',
+                    templateUrl: '$(Platform)/Scripts/common/templates/home.tpl.html',
+                    controller: [
+                        '$scope', 'platformWebApp.bladeNavigationService', function ($scope, bladeNavigationService) {
+                            var newBlade = {
+                                id: 'blade1',
+                                controller: '{ModuleId}.helloWorldController',
+                                template: 'Modules/$({Namespace})/Scripts/blades/hello-world.html',
+                                isClosingDisabled: true
+                            };
+                            bladeNavigationService.showBlade(newBlade);
+                        }
+                    ]
+                });
+        }
+    ])
+    .run(['$rootScope', 'platformWebApp.mainMenuService', 'platformWebApp.widgetService', '$state',
+        function ($rootScope, mainMenuService, widgetService, $state) {
```

Comment:
remove '$rootScope', it's never used in real modules

Reply:
fixed

Reply:
and also remove $rootScope